config/initializers/devise.rb

```diff
@@ -28,6 +28,9 @@
   # available as additional gems.
   require 'devise/orm/active_record'
 
+  config.omniauth :google_oauth2, 'GOOGLE_CLIENT_ID', 'GOOGLE_CLIENT_SECRET', {}
```

Comment:
Did you actually register the app as a google app and test it? You should have values for this. And use secrets and all that